### PR TITLE
updates to delete_sources & swarp

### PR DESCRIPTION
**Pull request by sidneymau**

### CODE REVIEW

File: eastlake/steps/delete_sources.py [hunk 1]
```diff
@@ -72,6 +72,16 @@ def execute(self, stash, new_params=None):
...
+                # ...and coadd object map
+                coadd_object_map_file = stash.get_filepaths(
+                    "coadd_object_map", tilename, band=band,
+                    keyerror=False,
+                )
+                if (coadd_object_map_file is not None):
+                    if os.path.isfile(coadd_object_map_file):
+                        self.logger.debug("removing file %s" % coadd_object_map_file)
+                        safe_rm(coadd_object_map_file)
+
```
Comment (sidneymau) [inline eastlake/steps/delete_sources.py:84]: as far as I know, this isn't needed for anything downstream (and is leftover from srcextractor/true detection processing...?)—assuming safe to delete

Comment (beckermr) [reply eastlake/steps/delete_sources.py:84]: We should run one sim with all of these recent PRs to make sure.

Comment (sidneymau) [reply eastlake/steps/delete_sources.py:84]: This wouldn't affect the sim as this is the last step after everything else is done. I think it's just a question of if the BFD folks need this particular file (from what I recall, they only mentioned needing the meds/fitvd stuff)

Comment (beckermr) [reply eastlake/steps/delete_sources.py:84]: Ah yeah we're good to go then.

**Pull request merged**